src/dataframerow/dataframerow.jl

```diff
@@ -14,7 +14,8 @@ DataFrameRow(parent::AbstractDataFrame, row::Integer, cols=:)
 ```
 
 A `DataFrameRow` supports the iteration interface and can therefore be passed to
-functions that expect a collection as an argument.
+functions that expect a collection as an argument. In such situations `eltype`
+of `DataFrameRow` is always `Any`.
```

Comment:
```suggestion
functions that expect a collection as an argument. Its element type is always `Any`.
```
That's always the case, so "in such situations" doesn't seem appropriate to me.)